diff --git a/libs/Common/Types.inl b/libs/Common/Types.inl
index 765db20..ccc52e4 100644
--- a/libs/Common/Types.inl
+++ b/libs/Common/Types.inl
@@ -3106,10 +3106,12 @@ static bool SaveImage(const cv::Mat& img, const String& fileName)
 		compression_params.push_back(cv::IMWRITE_JPEG_QUALITY);
 		compression_params.push_back(95);
 	} else
+#if CV_VERSION_MAJOR > 4 || (CV_VERSION_MAJOR == 4 && CV_VERSION_MINOR >= 11)
 	if (ext == ".jxl") {
 		compression_params.push_back(cv::IMWRITE_JPEGXL_QUALITY);
 		compression_params.push_back(95);
 	} else
+#endif
 	if (ext == ".pfm") {
 		if (img.depth() != CV_32F)
 			return false;
